src/cssModules.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
```

Comment:
I think authoring the files in JavaScript is quite nice here to avoid a build step entirely. Still, we can use TypeScript to validate the types in our modules.